src/js/main.js

```diff
@@ -548,7 +548,12 @@ function startProcess() {
     });
 
     ConfigStorage.get('darkTheme', function (result) {
-        setDarkTheme(result.darkTheme);
+        if (result.darkTheme === undefined) {
+            // sets dark theme to auto if not manually changed
+            setDarkTheme(2);
+        } else {
+            setDarkTheme(result.darkTheme);
```

Comment:
Minor niggle, but this will break on first load for users of 10.6.0-RC1, since `result.darkTheme` is `true` or `false`:

![image](https://user-images.githubusercontent.com/4742747/65595685-65372e80-dfe9-11e9-99c0-8b40f59de18c.png)

Reply:
Not sure how to work around that and no do something like if it is true or false then clear it or set it to auto. But that seems like a wrong thing to do.